kiteconnect/connect.py

```diff
@@ -11,7 +11,9 @@
 from six.moves.urllib.parse import urljoin
 import csv
 import json
-import dateutil.parser
+from dateutil.parser import parse
```

Comment:
you can import `parse` as `datetimeparse` or some other name which is explicit.

Reply:
Done.